client/my-sites/plugins/plugin-details-CTA/style.scss

```diff
@@ -16,34 +16,6 @@
 	.dialog__content {
 		padding: 0;
 	}
-
-	.eligibility-warnings {
```

Comment:
The goal here is to avoid specific eligibility-warnings code inside `plugin-details-CTA` to avoid duplicate code and simplify maintenance.